js/samples/ember-sample/package.json

```diff
@@ -66,15 +66,17 @@
     "edition": "octane"
   },
   "dependencies": {
-    "@microsoft/immersive-reader-sdk": "^1.2.0",
-    "@xmldom/xmldom": "^0.8.3",
+    "@microsoft/immersive-reader-sdk": "^1.3.0",
+    "@xmldom/xmldom": "0.8.4",
```

Comment:
should this be ^0.8.4?

Reply:
as expected, package was the security alert